src/wallet/feebumper.cpp

```diff
@@ -245,6 +245,19 @@ Result CreateRateBumpTransaction(CWallet& wallet, const uint256& txid, const CCo
         }
     }
 
+    // If no recipients, means that we are sending coins to a change address
+    if (recipients.empty()) {
+        // Just as a sanity check, ensure that the change address exist
+        if (std::get_if<CNoDestination>(&new_coin_control.destChange)) {
+            errors.emplace_back(Untranslated("Unable to create transaction. Transaction must have at least one recipient"));
+            return Result::INVALID_PARAMETER;
+        }
+
+        // Add change as recipient with SFFO flag enabled, so fees are deduced from it and no other output is created.
+        recipients.emplace_back(CRecipient{GetScriptForDestination(new_coin_control.destChange), output_value, /*fSubtractFeeFromAmount=*/true});
```

Comment:
In 7cb4dc157a711210e18f3c9b492150b6eb984b30 "bumpfee: enable send coins back to yourself"

`output_value` is not guaranteed to match what the user had actually specified for the single output.

Reply:
oops, fixed. And added coverage for it.